src/System.Management.Automation/engine/remoting/commands/StartJob.cs

```diff
@@ -512,10 +512,12 @@ public virtual Version PSVersion
 
             set
             {
-                RemotingCommandUtil.CheckPSVersion(value);
-
-                // Check if specified version of PowerShell is installed
-                RemotingCommandUtil.CheckIfPowerShellVersionIsInstalled(value);
+                // PSVersion value can only be 5.1 for Start-Job.
+                if (value.Major != 5 || value.Minor != 1)
```

Comment:
from a readability perspective, would it be more direct to the following:
```suggestion
                if (!(value.Major == 5 && value.Minor == 1))
```